src/async-binding.ts

```diff
@@ -42,6 +42,13 @@ export class asyncBindingBehavior {
             : () => { binding.originalupdateTarget(options && options.property ? this.getPropByPath(options.error, options.property) : options.error); }
           : undefined;
 
+        // Immediately update the target if the Observable has a BehaviorSubject-like current value member
+        if ("value" in a) {
+          binding.originalupdateTarget((a as any).value)
+        } else if (typeof (a as any).getValue === "function") {
+          binding.originalupdateTarget((a as any).getValue())

```

Comment:
does that really make a difference? I mean if you subscribe to a BehaviorSubject the subscription will fire right away with the current value.